laboratory-service/src/main/java/com/raf/si/laboratoryservice/controllers/LabExamController.java

```diff
@@ -0,0 +1,49 @@
+package com.raf.si.laboratoryservice.controllers;
+
+import com.raf.si.laboratoryservice.dto.request.CreateLabExamRequest;
+import com.raf.si.laboratoryservice.dto.request.UpdateLabExamStatusRequest;
+import com.raf.si.laboratoryservice.dto.response.LabExamResponse;
+import com.raf.si.laboratoryservice.service.LabExamService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.sql.Timestamp;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+@RestController
+@RequestMapping("/examination")
+public class LabExamController {
+    private final LabExamService labExamService;
+
+    public LabExamController(LabExamService labExamService) {
+        this.labExamService = labExamService;
+    }
+
+    @PreAuthorize("hasRole('ROLE_VISI_LAB_TEHNICAR')" + "or hasRole('ROLE_LAB_TEHNICAR')")
+    @PostMapping(path = "/create")
+    public ResponseEntity<LabExamResponse> createExamination(@Valid @RequestBody CreateLabExamRequest createLabExamRequest) {
+        return ResponseEntity.ok(labExamService.createExamination(createLabExamRequest));
+    }
+    @PreAuthorize("hasRole('ROLE_VISI_LAB_TEHNICAR')" + "or hasRole('ROLE_LAB_TEHNICAR')")
+    @GetMapping("/scheduled-count")
+    public ResponseEntity<Optional<Long>> getScheduledExamCount(@RequestParam("date") Timestamp date) {
```

Comment:
Timestamp moze lako da pukne ili da ne obradi ono sto se posalje kako treba. Treba postaviti format koji se ocekuje sa fronta
https://stackoverflow.com/questions/60948239/how-to-pass-timestamp-date-in-request-param-in-spring-boot-using-postman
Ovo moze da pomogne